packages/components/src/components/tabs/tabs.scss

```diff
@@ -1,22 +1,43 @@
 .dc-tabs {
-    &__list {
-        @include themify($themes) {
-            border-bottom: 1px solid themed('border_color');
+    position: relative;
+    @for $i from 1 through 4 {
+        .dc-tabs__item:nth-child(#{$i}).dc-tabs__active ~ .dc-tabs__active-line {
+            left: calc(var(--tab-width) * #{$i} - var(--tab-width));
         }
-        display: table;
+    }
+
+    &__list {
         width: 100%;
         line-height: 40px;
+        display: flex;
+        justify-content: center;
     }
     &__item {
-        display: table-cell;
         list-style: none;
         font-size: 14px;
         text-align: center;
+        padding-left: 24px;
```

Comment:
Please change these `24px` to `em` values?

Reply:
@mahdipourziaei-binary what em should i use? 2em?